provision/ansible/playbooks/templates/kube-vip-daemonset.yaml.j2

```diff
@@ -39,7 +49,9 @@ spec:
               value: kube-system
             - name: svc_enable
               value: "false"
-            - name: vip_address
+            - name: vip_ddns
+              value: "false"
+            - name: address
```

Comment:
I didn't read this change in the kube-vip changelog :(

Reply:
I see it was deprecated
https://github.com/kube-vip/kube-vip/blob/main/docs/flags/index.md